app/src/main/java/com/rentminder/service/MembersService.kt

```diff
@@ -9,7 +9,7 @@ import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
 class MembersService {
-    suspend fun fetchMembers() : List<Members>? {
+    suspend fun fetchMembersList() : List<Members>? {
```

Comment:
Changed function name to 'fetchMemberList()'. This makes it easier to understand what you are calling later on.